src/build/Dockerfile.in

```diff
@@ -18,15 +18,15 @@ COPY . /app/
 
 WORKDIR /app/source
 
+RUN rm -rf /app/source/cache/files/greenpeace/planet4-master-theme/
+
 RUN \
-    time composer -v update --lock --no-dev --no-ansi && \
-    time composer -v install --no-dev --no-ansi --no-interaction && \
     if [ -f "${SOURCE_PATH}/composer-local.json" ]; then \
       rm -f composer.lock; \
-      time composer -v config --no-ansi extra.merge-plugin.require composer-local.json; \
-    fi && \
-    time composer -v update --lock --no-dev --no-ansi && \
-    time composer -v install --no-dev --no-ansi --no-interaction
```

Comment:
Tried to understand the double install, the original commit https://github.com/greenpeace/planet4-builder/commit/1ba0ee9c3890807da95fd2e43524aa3a66582422 talks about fixing #3 which I couldn' find, but while typing this the autocompletion might have given a hint (still don't know if it makes sense :shrug: maybe issues were open in this repo at some point ?)
![issue3](https://user-images.githubusercontent.com/617346/94662919-59233280-0309-11eb-992b-c48f56ecd721.png)

Reply:
Nice find :detective: Yeah I think I'm going to leave it as is, unless changing it would be needed to get the zip release plugin to work. There's too many possible edge cases if we change the behavior, so if we do it's better in a dedicated PR.

We could look into fetching the shared part of the dependencies in the `planet4-base-fork` workflow so that NROs just need to update what's different for them.

Also it doesn't really seem to affect performance as the 3 calls after the first `update` are done within seconds :shrug: 

I'll still split it up in multiple `RUN` statements to improve the output, instead of chaining them.

Reply:
Re-enabled issues temporarily on the repo, so we can have a sneak peak at #3 :eyes:

Reply:
So there was a `Your requirements could not be resolved to an installable set of packages.` error after merging composer files, and that did not stop the process, I thought that was an exit code 2 which would stop everything. The merge-plugin might be [masking some error](https://github.com/wikimedia/composer-merge-plugin/issues/180)

Reply:
Looks like removing the cache might not be needed. I made another branch where I preserved the composer commands (though in separate `RUN` layers instead of chained with `&&`, but that should work the same). I only added removing the cache there before the composer commands, but it didn't cause it to fetch the zip. 
https://github.com/greenpeace/planet4-builder/compare/planet-5484-only-remove-cache

Then I added `--prefer-dist` to all commands, and it did work, even though the version with the zip of theme and plugin is only installed after configuring the local composer. So I expect the plugin is dealing correctly with the cache after all.

I then dropped the line that removes the cache again and reran the test instance workflow.
https://app.circleci.com/pipelines/github/greenpeace/planet4-test-iocaste/39/workflows/ff00dcf5-befa-4616-8c03-326057ec39c5/jobs/277

It actually was able to re-use the cache from [a previous workflow](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-iocaste/39/workflows/50bd6c72-74f4-4e82-8dce-c7a82b497e04/jobs/270) which correctly contained the github zip release code. I'll test a bit more to be sure it keeps handling it correctly.